File: bmi_calculator.c
#include <stdio.h>

int main() {
    float weight, height_cm, height_m, bmi;

    // Get weight in kilograms
    printf("\nEnter your weight in kilograms: ");
    scanf("%f", &weight);

    // Get height in centimeters
    printf("Enter your height in centimeters: ");
    scanf("%f", &height_cm);

    // Convert height from centimeters to meters
    height_m = height_cm / 100;

    // Calculate BMI
    bmi = weight / (height_m * height_m);

    // Print the BMI
    printf("\nYour Body Mass Index (BMI) is: %.2f\n", bmi);

    // Interpret BMI
    if (bmi < 18.5) {
        printf("Category: Underweight\n\n");
    } else if (bmi < 25) {
        printf("Category: Normal weight\n\n");
    } else if (bmi < 30) {
        printf("Category: Overweight\n\n");
    } else {
        printf("Category: Obese\n\n");
    }

    return 0;
}

File: bmi_calculator.py
# Get weight in kilograms
weight = float(input("\nEnter your weight in kilograms: "))

# Get height in centimeters
height_cm = float(input("Enter your height in centimeters: "))

# Convert height from centimeters to meters
height_m = height_cm / 100

# Calculate BMI
bmi = weight / (height_m ** 2)

# Print the BMI
print(f"\nYour Body Mass Index (BMI) is: {bmi:.2f}")

# Interpret BMI
if bmi < 18.5:
    print("Category: Underweight\n")
elif bmi < 25:
    print("Category: Normal weight\n")
elif bmi < 30:
    print("Category: Overweight\n")
else:
    print("Category: Obese\n")